src/accelerator.rs

```diff
@@ -0,0 +1,98 @@
+use drivers::Driver;
+use Motion;
+use Operation;
+
+pub struct Accelerator<D: Driver> {
+    motions: Vec<Motion>,
+    driver: D,
+}
+
+impl <D: Driver> Accelerator<D> {
+    pub fn new(mots: Vec<Motion>, drv: D) -> Self {
+        Accelerator{motions: mots, driver: drv}
+    }
+    // Needs to be un pub.ed
+    pub fn within_bounds(&self, n: isize) -> isize{
+        println!("called bounds with \'{}\'", n);
```

Comment:
No.. just the best way I currently know + I needed something so that I could see it was doing things. Will be removed.